### DISCUSSION TITLE
Converting q-table example code <basic usage> to ts

### DISCUSSION THREAD

**Discussion by Sashley** [quasarframework/quasar#13146] [General - Components / Directives / etc] [answered]
Play with converting 'basic usage' example code 
  i.e. https://codepen.io/sashley-the-typescripter/pen/gOojXwO?editors=101
for q-table to typescript but stumbling with types for field and format functions 
  i.e. field: (row: object) => row.name returning a TS(2503) error failed unable to find namespace 'row' and 
  format: (val: string) => `${val}` returning a TS(2749) error with 'val' refers to value but used as type. 

Am struggling with this, so any hints/ advice on how to fix them?

**Comment by metalsadman** [comment, marked as answer]
@Sashley please check the pinned messages in our `v2-typescript` discord channel. https://discord.com/channels/415874313728688138/807654245640962059/949981766448447489
